doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,66 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `ServerOptions` and `ClientOptions`
+
+The two most important settings are:
+
+| Option                                 | Explanation                                                                  |
+|----------------------------------------|------------------------------------------------------------------------------|
+| `ServerOptions::requestQueueCapacity`  | This enables servers to store n requests before they are passed to the users |
+| `ClientOptions::responseQueueCapacity` | This enables clients to store n respones before they are passed to the users |
```

Comment:
I like `respones` ;)

Also, technically they are already at the user side. How about

```suggestion
| `ClientOptions::responseQueueCapacity` | This enables clients to store n responses at the users side |
```

Could you also change this for server and subscriber?